cmd/ncproxy/hcn_networking_test.go

```diff
@@ -987,8 +999,9 @@ func TestAddEndpoint_NoError(t *testing.T) {
 			}()
 
 			req := &ncproxygrpc.AddEndpointRequest{
-				Name:        endpointName,
-				NamespaceID: namespace.Id,
+				Name:         endpointName,
+				NamespaceID:  namespace.Id,
+				AttachToHost: test.attachToHost,
```

Comment:
We change the namespace id in the implementation when `AttachToHost` is set, so wouldn't the below check that the endpoint was added to the namespace fail in that case?

Reply:
It passes (atleast, locally ...)
I think cause for the tests we create the test namespace in the default namespace, so the names are the same

Should i switch the test to creating a new non-`HostDefault` namespace for the endpoints then update this check?

Reply:
hmm yeah I think we should switch it

Reply:
updated